### PR TITLE
MEALS TOKEN

### PR DESCRIPTION
**Pull request by MEALS-TOKEN**
# Validate  [MEALS](https://solscan.io/token/BLimDFGYnr6SNNWEGiXbe2FYSKgyHhjbVKMt47h5GezV)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/MealsToken/status/1765982804113043518
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by AustinArrowood — COMMENTED** [review]
I need help finding 2 reviews. So confused here . https://github.com/AustinArrowood/MEALS-TOKEN/tree/main

**Comment by AustinArrowood** [comment]
I need reviews. How do I get those? Confused here. https://github.com/AustinArrowood/MEALS-TOKEN/tree/main

**Review by Mogzix85 — COMMENTED** [review]
Good

**Review by Mogzix85 — COMMENTED** [review]
Looks great!!

File: validated-tokens.csv [hunk 1]
```diff
@@ -1162,6 +1162,7 @@ VICE,VICE,AWeL8nCXxJm3GZkEVRqxeeT5KWT2KwDa1shgQFeBogkC,9,https://bafybeieprtdo3x
 Nyxia Coin,NYXC,HrLx8MLKegpbmbmWePZiuvf3AbJNx1CJyjBwUHwicEgW,7,http://nyxia.ai/NYXC_coin-200.png,true
 pepewifhat,PIF,8vRnnknwERunJEEDtEFoxzaxbxGnEY61FPbx8uTVKsUD,6,https://v2.akord.com/public/vaults/active/crc1z0v2OurbSFWHPx6khAKm_ui8_NLAHl3Ho_SgsGU/gallery#2a08f614-9d37-4dec-bfe6-1e4b93123ff4,true
 PopDog,POPDOG,EATGZHJViJsk7nEKkrdJicwNbfpkJfAtmrEmrjXR8NBj,6,https://i.postimg.cc/SRb6fSps/SBZ5KM3L.png,true
+MEALS TOKEN,MEALS,EYgtMnEE2c1Yyi8PwJTz2gPN3twHQXQr9YsxRkwWBW7q,5,https://bafybeie4hovztz2rvqbip7d6jvudo7jpfodmmnyrkhlw2rzh5iafhurdum.ipfs.nftstorage.link/,true
```
Comment (Misterglut) [inline validated-tokens.csv:1165]: That's the one, $MEALS deserves to be added to Jupiter strict list. CA matches

**Comment by oyoops** [comment]
> I need help finding 2 reviews. So confused here . https://github.com/AustinArrowood/MEALS-TOKEN/tree/main

The 2 reviews come from Jupiter list admins. Expect to wait until day 21 since your coin launched. Cheers... and I'm sorry about the sale.

**Comment by MEALS-TOKEN** [comment]
> > I need help finding 2 reviews. So confused here . https://github.com/AustinArrowood/MEALS-TOKEN/tree/main
> 
> The 2 reviews come from Jupiter list admins. Expect to wait until day 21 since your coin launched. Cheers... and I'm sorry about the sale.



> > I need help finding 2 reviews. So confused here . https://github.com/AustinArrowood/MEALS-TOKEN/tree/main
> 
> The 2 reviews come from Jupiter list admins. Expect to wait until day 21 since your coin launched. Cheers... and I'm sorry about the sale.

@oyoops Ohhh I see hahaa. They have to have write access. Thanks for the heads up

**Comment by Misterglut** [comment]
- [x] Mint address: BLimDFGYnr6SNNWEGiXbe2FYSKgyHhjbVKMt47h5GezV

- [x]  Symbol: MEALS

- [x]  Token Decimals: 5

- [x]  Token icon: https://bafkreihz34bzin2seyxgx5z336kpun5kkdn6huxy3hro264zddjayjnzzu.ipfs.nftstorage.link

- [x]  On-chain metadata matches

- [x]  No duplication on Jupiter's strict list

**Pull request closed without merging**